app/javascript/app/components/ndcs-country-accordion/ndcs-country-accordion-styles.scss

```diff
@@ -0,0 +1,53 @@
+@import '~styles/layout.scss';
+
+$min-height: 450px;
+
+.wrapper {
+  position: relative;
+  min-height: $min-height;
+}
+
+.loader {
+  min-height: $min-height;
+  position: absolute;
+  top: 0;
+  right: 0;
+  left: 0;
+  z-index: 5;
+}
+
+.noContent {
+  font-size: $font-size-sm;
+  color: $dark-gray;
+  font-style: italic;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  min-height: $min-height;
+}
+
+.title {
+  justify-content: center;
+}
+
+.mainTitle {
+  @extend .title;
+  text-transform: uppercase;
+}
+
+.secondaryTitle {
+  @extend .title;
+}
+
+.secondaryHeader {
+  background-color: $light-gray;
```

Comment:
Álvaro, I think this color should be set to `#ECF1FA` as it is here: https://projects.invisionapp.com/d/main#/console/15950314/330875045/inspect
It's a bit lighter and so harsh on the eyes